GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_Utility.cfg

```diff
@@ -0,0 +1,100 @@
+@PART[M2X_AligningDockingPort|M2X_Mk2BatteryBank|M2X_cargoContainer|M2X_ShieldedDockingPort|M2X_SmallLab|M2X_RadialLeg|M2X_Nosebay|M2X_PoddedRadiator|M2X_Reactor|M2X_Servicebay|M2X_SolarpanelPod]:FOR[RealismOverhaul]
+{
+	%skinMaxTemp = 2500
+}
+
+@PART[M2X_AligningDockingPort]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.5
+	rescaleCube = 1.722222
+	@DRAG_CUBE
+	{
+		rescaleX = 1.722222
+		rescaleY = 1.722222
+		rescaleZ = 1.722222
+	}
+	@MODULE[ModuleDockingNode]
+	{
+		@nodeType = NASADock
+		%acquireForce = 0.5 // 2
+		%acquireMinFwdDot = 0.8 // 0.7
+		%acquireminRollDot = -3.40282347E+38
+		%acquireRange = 0.25 // 0.5
+		%acquireTorque = 0.5 // 2.0
+		%captureMaxRvel = 0.1 // 0.3
+		%captureMinFwdDot = 0.998
+		%captureMinRollDot = -3.40282347E+38
+		%captureRange = 0.05 // 0.06
+		%minDistanceToReEngage = 0.25 // 1.0
+		%undockEjectionForce = 0.1 // 10
+	}
+}
+
+@PART[M2X_Mk2BatteryBank]:FOR[RealismOverhaul]
```

Comment:
Only holding 2400 EC isn't much. This should probably just be configured as a ServiceModule, or not configured at all.

Reply:
That's not what I suggested.